docs/control-structures.rst

```diff
@@ -29,8 +29,13 @@ something like::
         }
     }
 
-Input parameters can be used just as any other local variable
-can be used, they can also be assigned to.
+Input parameters can be used as any other local variable and they can also be assigned to.
+
+.. note::
+    A function cannot accept a multi-dimensional array as an input parameter.
+    This functionality is possible if you enable the new experimental ``ABIEncoderV2``
+    feature by adding ``pragma experimental ABIEncoderV2;`` to your source file.
+    This feature introduces other experimental changes that you can read more about in the :ref:`ABI` guide.
```

Comment:
I don't think the "ABI" section explains what features ABIEncoderV2 introduces.

Reply:
@axic In that case, I'm not aware of anywhere in the docs that does. Is there anywhere else I can find the content to get it included?

Reply:
Unfortunately, there is no section that covers the ABIEncV2 exclusively. We could remove the last sentence for now, create that subsection with another PR, grep through all the places where the pragma is used and then link to that section.